### PR TITLE
Add WanNianLi app

### PR DESCRIPTION
**Pull request by asins**
After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/wan-nian-li.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "wan-nian-li" do
+  version :latest
+  sha256 :no_check
+
+  url "https://calendar.zfdang.com/WanNianLi.app.zip",
+      verified: "calendar.zfdang.com"
+  name "WanNianLi"
+  desc "Mac下的万年历状态栏小工具"
+  homepage "https://github.com/zfdang/chinese-lunar-calendar-for-mac/"
```
Comment (vitorgalvao) [inline Casks/wan-nian-li.rb:9]: Homepage seems to be http://calendar.zfdang.com.

File: Casks/wan-nian-li.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  depends_on macos: ">= :el_capitan"
+
+  app "WanNianLi.app"
+
+  zap trash: [
```
Comment (vitorgalvao) [inline Casks/wan-nian-li.rb:15]: If it’s only one item it can be a string, no need to be an array.

File: Casks/wan-nian-li.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+cask "wan-nian-li" do
+  version :latest
+  sha256 :no_check
+
+  url "https://calendar.zfdang.com/WanNianLi.app.zip",
+      verified: "calendar.zfdang.com"
+  name "WanNianLi"
+  desc "Mac下的万年历状态栏小工具"
```
Comment (vitorgalvao) [inline Casks/wan-nian-li.rb:8]: Description must be in English.

File: Casks/wan-nian-li.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+cask "wan-nian-li" do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/wan-nian-li.rb:2]: Please add a `version` and `sha256`.

File: Casks/wan-nian-li.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  sha256 :no_check
+
+  url "https://calendar.zfdang.com/WanNianLi.app.zip",
```
Comment (vitorgalvao) [inline Casks/wan-nian-li.rb:5]: Use the GitHub releases as downloads and we’ll get versions and `livecheck` for free.

File: Casks/wan-nian-li.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
+cask "wan-nian-li" do
```
Comment (vitorgalvao) [inline Casks/wan-nian-li.rb:1]: Does not follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md). There must be no spaces.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/wan-nian-li.rb (hunk 4 above)
Comment (asins) [reply Casks/wan-nian-li.rb:2]: This is my original writing:
 version "2019-12-06"
 sha256 "702298f34ca2576a02388a4103fc0e09a4aae753df21ca2e012f6f87497db6e9"

 But an error is reported when executing `brew audit --cask wan-nian-li`:
```bash
audit for wan-nian-li: failed
 - Use `sha256 :no_check` when URL is unversioned.
Error: 1 problem in 1 cask detected
```

File: Casks/wan-nian-li.rb (hunk 6 above)
Comment (asins) [reply Casks/wan-nian-li.rb:1]: wan-nian-li comes from Chinese pinyin `Wan Nian Li`, I use `-` as the connector for each word, there is no space.

File: Casks/wan-nian-li.rb (hunk 5 above)
Comment (asins) [reply Casks/wan-nian-li.rb:5]: The issue has been submitted to the software author, and I will update it when github releases are provided.

**Comment by asins** [comment]
All modified

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you!

**Pull request merged**